package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "frequency-style-guide",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "src/index.ts",
+  "types": "src/index.d.ts",
+  "scripts": {
+    "storybook": "storybook dev -p 6006 -s public",
+    "build-storybook": "storybook build",
+    "publish-package": "npm publish",
```

Comment:
This seems unnecessary, since you could just run `npm publish` instead of `npm run publish-package`
```suggestion
```